compiler/rustc_passes/src/check_attr.rs

```diff
@@ -303,6 +303,16 @@ impl CheckAttrVisitor<'tcx> {
                                 .emit();
                             return false;
                         }
+                        if doc_alias.starts_with(' ') || doc_alias.ends_with(' ') {
```

Comment:
What if people put `\t`?

Reply:
`\t` is handled above on line 292, it's not allowed at all.